classes/class3/homework/routes/index.js

```diff
@@ -0,0 +1,74 @@
+var mongoose = require('mongoose');
+
+var mongoURI = process.env.MONGOURI || "mongodb://localhost/test";
+mongoose.connect(mongoURI);
+
+var catSchema = mongoose.Schema({
+	name: String,
+	age: Number,
+	colors: [String]
+});
+
+var Cat = mongoose.model('Cat',catSchema);
+
+var names = ['Bob','Jack','Sandy','Sasha','Brendan','Phillly','Jen','Flufferton','Oprah'];
+var colopts = ['red','yellow','black','white','brown','orange','gray'];
+
+var home = function(req, res){
+	res.render("home", {"pages": [
+	  'create a new cat',
+	  'show all cats',
+	  'show colored cats',
+	  'delete old cat']
+	});
```

Comment:
It would have been nice for these to be links instead of just list items, not a big deal though.